docs/tiling.md

```diff
@@ -0,0 +1,277 @@
+    "tilejson": "3.0.0",
+    "name": "2D Data Example",
+    "description": "A tileset showing 2D data with multiple layers of detail.",
+    "version": "1.0.0",
+    "attribution": "<a href='http://example.com'>Example</a>",
+    "tiles": [
+        "http://example.com/tiled_example/{zlvl}/{x}/{y}.json"
+    ],
+    "minzoom": 0,
+    "maxzoom": 10,
+    "bounds": [0, 0, 24000, 24000],
+    "center": [12000, 12000, 0],
+    "vector_layers": [
+        {
+            "id": "image_layer",
+            "description": "Image layer",
+            "minzoom": 0,
+            "maxzoom": 10,
+            "fields": {
+                "string": {
+                    "plate": "Plate ID",
+                    "image": "Image URI"
+                },
+                "numerical": {
+                    "label": "Label ID",
+                    "channel": "Channel ID"
+                }
+            }
+        }
+    ],
+    "fillzoom": 3
+}
+```
+
+## Tiled MicroJSON Example
+### Level 0
+The following is an example of a MicroJSON object at zoom level 0, tile at (0,0) in the tiling scheme. Example URL: `http://example.com/tiles/0/0/0.json`
+```json
+{
+    "tilejson": "3.0.0",
+    "name": "2D Data Example",
+    "description": "A tileset showing 2D data with multiple layers of detail.",
+    "version": "1.0.0",
+    "attribution": "<a href='http://example.com'>Example</a>",
+    "tiles": [
+        "http://example.com/tiled_example/{zlvl}/{x}/{y}.json"
+    ],
+    "minzoom": 0,
+    "maxzoom": 10,
+    "bounds": [0, 0, 24000, 24000],
+    "center": [12000, 12000, 0],
+    "format": "json",
+    "vector_layers": [
+        {
+            "id": "image_layer",
+            "description": "Image layer",
+            "minzoom": 0,
+            "maxzoom": 10,
+            "fields": {
+                "string": {
+                    "plate": "Plate ID",
+                    "image": "Image URI"
+                },
+                "numerical": {
+                    "label": "Label ID",
+                    "channel": "Channel ID"
+                }
+            }
+        }
+    ],
+    "fillzoom": 3
+}
+```
+
+### Level 1 
+The following is an example of a MicroJSON object at zoom level 1, tile at (0,1) in the tiling scheme. Example URL: `http://example.com/tiles/1/0/1.json`
+```json
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [
+              0,
+              10000
+            ],
+            [
+              10000,
+              10000
+            ],
+            [
+              10000,
+              20000
+            ],
+            [
+              0,
+              20000
+            ],
+            [
+              0,
+              10000
+            ]
+          ]
+        ]
+      },
+      "properties": {
+        "numeric": {
+          "label": 3
+        }
+      }
+    }
+  ],
+  "multiscale": {
```

Comment:
"multiscale" multiScale

Reply:
multiscale without capitalization as in camelCase comes from OME NGFF metadata model.